MockMDS/src/main/java/io/mosip/mock/sbi/service/SBIServiceResponse.java

```diff
@@ -155,7 +155,7 @@ public String processDeviceDicoveryInfo (SBIMockService mockService)
         	 DeviceDiscoveryRequestDetail requestObject = (DeviceDiscoveryRequestDetail) getRequestJson (SBIConstant.MOSIP_DISC_VERB);
         	 String type = null;
         	 if (requestObject != null && requestObject.getType() != null && requestObject.getType().length() > 0)
-        		 type = requestObject.getType().toString ().trim().toLowerCase();
+        		 type = requestObject.getType().toString ().trim();

```

Comment:
Is there any issue in converting to lowercase?

Reply:
Not in getType, we have reverted this and removed the same toLowerCase() on getDeviceStatus() method. As further comparisons of device status are case-sensitive.

Reply:
109 error Requested number of biometric (Finger/IRIS) not supported already defined in the spec which is required to test in complaince toolkit